pkgs/development/ruby-modules/runtests.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -o xtrace
+pwd
```

Comment:
Just a single `pwd` output is impossible to find. Prefix it with something one can grep for or remove it.